doc/docs/master/concepts/pipeline-concepts/datum/relationship-between-datums.md

```diff
@@ -1,164 +1,86 @@
 
-## Note: Data persistence between datums
+The content of all `/pfs/out` is then combined in a commit to the pipeline's output repo. 
+This generally means unioning all the files together.
+
+!!! Important "The Single Datum Provenance Rule"
+     If two outputted files have the same name (i.e. two datums wrote to the same output file, creating a conflict), then an error is raised resulting in your pipeline failure. 
+     An easy way to avoid this anti-pattern from the start is to **have each datum write in separate files.
+### 5. **Next: Add a `Reduce` pipeline**
+
+If you need files from different datums merged into single files in a particular way, **add a pipeline that groups the files from the previous output repo into single datums using the appropriate glob pattern and merges them as intended using your code**. The example that follows illustrates this two steps approach. 
+
+### Example: 2 Steps pattern and Single Datum Provenance Rule
+
+In this example, we highlight a 2 pipelines pattern where a first pipeline's glob pattern splits an incoming commit into 3 datums (called "Red", "Blue", "Purple"), each producing 2 files each.
+The files can then be further
+appended or overwritten with other files to create your final result. In the example below, the following pipeline appends the content of all files in each directory into one final document.
 
-Pachyderm only controls and wipes the /pfs directories between datums. If scratch/temp space is used during execution, the user needs to be careful to clean that up. Not cleaning temporary directories may cause unexpected bugs where one datum accesses temporary files that were previously used by another datum!
+
+!!! Note "Worth Noting"
+    - In the example, the files are named after the datum itself. Depending on your use case, there might be more logical ways to name the files produced by a datum. However, in any case, make sure that this **name is unique for each datum** to avoid duplicate
+    files with the same file path.
+    - Each file is put in specific directories. This directory structure has been thought to facilitate the aggregation of the content in the following pipeline. Think about your directory structure so that the next glob pattern will aggregate your data as needed.
+
+
+![Map Reduce](../../../images/parallel_data_processing.png)
+
+
+Let's now create a new commit and overwrite a file in `datum 2`,
+Pachyderm detects three datums. However, because `datum 1` and `datum 3` are
+unchanged, it skips processing these datums. Then, Pachyderm detects
+that something has changed in `datum 2`. Pachyderm is unaware of any
+details of the change; therefore, it processes the whole `datum 2`
+and outputs 3 files. Then, the following pipeline aggregates
+these data to create the following final result:
+
+![Map Reduce](../../../images/parallel_data_processing_following_commit.png)
+
+## Incrementality 
+In Pachyderm, glob patterns are applied to the entire input, 
```

Comment:
Shouldn't mention glob patterns here.

Reply:
Removed `glob patterns are applied to the entire input,`